index.html

```diff
@@ -0,0 +1,129 @@
+            </svg><p>facebook</p>
+        </div>
+    </header>
+    <nav class="nav">
+        <div class="my-logo">
+            <img class="logo-img" src="./assets/resto_logo.jpg" alt="resto_logo">
+            <p class="resto-logo-name">Isekai restaurant</p>
+        </div>
+        <div class="container" id="hamburger">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>
+        <ul id="myLinks">
+            <li>
+                <a href="#news">News</a>
+            </li>
+            <li>
+                <a href="#categories">Some categories</a>
+            </li>
+            <li>
+                <a href="about.html">About Us</a>
+            </li>
+        </ul>
+    </nav>
+    <section class="intro">
+        <!-- <img class="intro-img" src="./assets/resto1.jpg" alt="intro-img"> -->
+        <div class="intro-content">
+            <h2 class="intro-message">"Hello you there come and eat"</h2>
+            <h1>Welcome to Isekai Restaurant</h1>
+        </div>   
+    </section>
+    <section class="news-events" id="news">
+        <h2>News & Events</h2>
+        <div class="line"></div>
+        <div class="news col-12 col-lg-10 d-md-flex justify-content-center">
+            <div class="card col-10 mx-auto col-md-3 m-3 d-flex d-md-block flex-row" >
+                <img src="./assets/steak.jpeg" class="card-img-top" alt="steak">
+                <div class="card-body w-100">
+                    <h5 class="card-title">Comming Soon</h5>
+                    <p class="card-text">Minima aperiam ullam veniam fugiat asperiores sed qui, voluptatum eius? Quisquam </p>
+                    <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
+                </div>
+            </div>
+            <div class="card col-10 mx-auto col-md-3 m-3 d-flex d-md-block flex-row" >
+                <img src="./assets/resto3.jpg" class="card-img-top" alt="resto">
+                <div class="card-body w-100">
+                    <h5 class="card-title">News</h5>
+                    <p class="card-text"> Minima aperiam ullam veniam fugiat asperiores sed qui, voluptatum eius? Quisquamat!</p>
+                    <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
+                </div>
+            </div>
+            <div class="card col-10 mx-auto col-md-3 m-3 d-flex d-md-block flex-row" >
+                <img src="./assets/decert-cup.jpg   " class="card-img-top" alt="decert">
+                <div class="card-body w-100">
+                    <h5 class="card-title">Event</h5>
+                    <p class="card-text"> Minima aperiam ullam veniam fugiat asperiores sed qui, voluptatum eius? Quisquam </p>
+                    <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
+                </div>
+            </div>
```

Comment:
- [ ] You're expected to have 5 articles here in desktop view but you currently have 3. Kindly add 2 more

Yours | Orignal
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124146605-d7258d00-da85-11eb-84ba-55928c54b1f8.png) | ![image](https://user-images.githubusercontent.com/16058470/124146669-e6a4d600-da85-11eb-8c3f-f410bc7045a1.png)



- [ ] On mobile view, this section should be organized in a vertical column like it is in the design below.
- [ ] The individual articles should have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124149562-9aa76080-da88-11eb-952b-ea048b1a6747.png) | ![image](https://user-images.githubusercontent.com/16058470/124149651-b3177b00-da88-11eb-8102-5dbda2cefd48.png)

Reply:
ok i fixed it. hope its fine now